### DISCUSSION TITLE
Hosting a server

### DISCUSSION THREAD

**Discussion by FunctionalMetatable** [ScratchAddons/ScratchAddons#2695] [Discussion]
FossHost may be able to host a server for us so we could make isOnline and more addons a reality.

**Comment by davwat** [comment]
interesting idea, sa is probably big enough, but it is still technically a "centralized server" so i'm not sure how that would go down...

**Reply by FunctionalMetatable** [reply]
but then it's a "communal effort"

**Reply by davwat** [reply]
pinging @WorldLanguages to see what they think.

**Comment by qbjl** [comment]
There's also something called node.js and mysql

**Reply by FunctionalMetatable** [reply]
we get the server from fosshost and use node on it

**Reply by davwat** [reply]
also sql is aaaaa, use mongodb
